src/Engine/Entity/Entity.cpp

```diff
@@ -336,6 +334,9 @@ unsigned int Entity::GetUniqueIdentifier() const {
     return uniqueIdentifier;
 }
 
+void Entity::SetDirection(glm::vec3 rot) {
+    rotation = rot;
```

Comment:
There has been other changes to `GetDirection` which returns the direction in world space. We should be consistant and set world space as well. As of now this method sets the direction in the local space of the entity.